src/content/autoBlock.ts

```diff
@@ -118,6 +118,13 @@ interface User {
 	alreadyBlocked: boolean;
 }
 
+function blockEthUsername(userName?: string) {
+	console.log(userName);
+	return !settings?.actionOnEthUsernames || !userName
+		? false
+		: new RegExp('.eth', 'gi').test(userName);
```

Comment:
This would block "beth" because of the `.`. You want `/\.eth$/`, probably

Reply:
lol ops - should go unblock a few elizabeths, bethanys, and beths 😂